addons/web/static/src/core/l10n/dates.js

```diff
@@ -69,6 +69,34 @@ export function areDateEquals(d1, d2) {
     return d1 instanceof DateTime && d2 instanceof DateTime ? d1.equals(d2) : d1 === d2;
 }
 
```

Comment:
add info on the fact that sunday is the first day or not according to localization (main point of the commit actually)